repository/release.go

```diff
@@ -36,6 +37,10 @@ func (r *ReleaseRepository) Create(ctx context.Context, rls svcmodel.Release) er
 		"updatedAt":    rls.UpdatedAt,
 	})
 	if err != nil {
+		if util.IsUniqueConstraintViolation(err, "unique_release_title_per_project") {
```

Comment:
In the case of `releases` entity, a unique constraint violation can occur in two situations:

- Inserting a duplicated primary key (which, in the case of UUIDs, is highly unlikely in the real world of course).
- Inserting a duplicated release title.

Therefore, it would be acceptable to simply catch the unique constraint error without checking the specific constraint and return a `NewReleaseDuplicateTitleError`. However, I believe it is better to make the handling more explicit.

Reply:
OK, I like the explicitness.
But then perhaps the name of the constraint deserves to be a constant.

Reply:
(or simply returning a more general `ALREADY_EXISTS` error - also fine)